docs/0-zarf-overview.md

```diff
@@ -59,60 +59,66 @@ For more technical information on how Zarf works and to view the Zarf architectu
 
 For additional information, see the [Creating a package](./5-zarf-tutorials/0-creating-a-zarf-package.md) section.
 
 ### (2) Ship the Package to the System Location
 
-Zarf enables secure software delivery for various environments, such as remote, constrained, independent, and air-gapped systems. Considering there are various target environments with their own appropriate transferring mechanisms, Zarf does not determine _how_ packages are moved so long as they can arrive in your downstream environment.
+Zarf enables secure software delivery for various environments, such as remote, constrained, standalone, and air-gapped systems. Considering there are various target environments with their own appropriate transferring mechanisms, Zarf does not determine _how_ packages are moved so long as they can arrive in your downstream environment.  See [Package Sources](./4-deploy-a-zarf-package/2-package-sources.md) for more information on where Zarf packages can be stored / pulled from.
 
 ### (3) Deploy the Package
 
 Once your package has arrived, you will need to:
 
 1. Install the binary onto the system.
-2. Run the zarf init package.
+2. Initialize a cluster with a zarf init package.
 3. Deploy the package to your cluster.
 
 ## Cluster Configuration Options
 
-Zarf allows the package to either deploy to an existing K8s cluster or a local K3s cluster. This is a configuration that is available on deployment in the init package.
+Zarf allows the package to either deploy to an existing K8s cluster or a K3s cluster it creates. This configuration is available on deployment of the init package.
 
-### Appliance Cluster Mode
+### As an Appliance
 
-![Appliance Mode Diagram](.images/what-is-zarf/appliance-mode.png)
+![Appliance Cluster Diagram](.images/what-is-zarf/appliance.drawio.png)
 
-In the simplest usage scenario, your package consists of a single application (plus dependencies) and you configure the Zarf cluster to serve your application directly to end users. This mode of operation is called "Appliance Mode" and it is intended for use in environments where you want to run K8s-native tooling but need to keep a small footprint (i.e. single-purpose/constrained/"Edge" environments).
+In the simplest usage scenario, you deploy the Zarf init package's builtin cluster and use it to serve your application(s) directly to end users. This configuration runs Zarf and it's init package components as a self-contained appliance and is intended for use in environments where you want to run K8s-native tooling but need to keep a small footprint (i.e. single-purpose/constrained/"Edge" environments).
 
-### Utility Cluster Mode
+### With an Existing Cluster
```

Comment:
This change breaks the pattern previously established of using mode name and moving to a description of the type of cluster it creates (but the one above does not follow this new descriptive pattern. I think they should either both be feature names or both describe the type of cluster

Reply:
Is this better now?

Reply:
Yes